### DISCUSSION TITLE
Draw icons on map

### DISCUSSION THREAD

**Discussion by brandonmcclure** [fyne-io/fyne-x#96] [Q&A]
I have followed the map example to add a map next to a table in my layout. It is very cool how quick it is to get going with these widget in a nice layout. The table data gets updated when a slice containing bus stop data is updated. This stop struct contains properties for latitude and longitude which I would like to display on my map. It is not clear to me how I should approach this though. 

I looked around for a property of the Map widget looking for a "markers" collection that I would need to add/update. There are some MapOptions, but those do not seem to be useful for what I am trying to do. Could someone help point me to the right direction for drawing a list of geographic coordinates inside of the map widget?

**Comment by andydotxyz** [comment]
We have not added that feature yet, so it would be good to get feedback on how you think it should work.

As this is a Fyne widget you can easily draw over it using `container.NewStack` to place your area over the map. However we need to share with you the current location (/bounds) of the map, and when it changes. Let's collaborate to get this in!
